### PR TITLE
Changelog for v1.15.7

### PR DESCRIPTION
**Pull request by lunny**
blocked by #17882, and #17881 needs to be added

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -4,6 +4,37 @@ This changelog goes through all the changes that have been made in each release
...
+  * Fix bug when read mysql database max lifetime (#17682) (#17690)
+  * Fix database deadlock when update issue labels (#17649) (#17665)
+  * Fix bug on detect issue/comment writer (#17592)
+  * Remove appSubUrl from pasted images (#17572) (#17588)
+  * Make `ParsePatch` more robust (#17573) (#17580)
+  * Fix stats upon searching issues (#17566) (#17578)
+  * Escape issue titles in comments list (#17555) (#17556)
+  * Fix zero created time bug on commit api (#17546) (#17547)
+  * Fix database keyword quote problem on migration v161 (#17523)
+  * fix email with + when active (#17518) (#17520)
```
Comment (techknowlogick) [inline CHANGELOG.md]: Please capitalize "Fix"

Comment (lunny) [reply CHANGELOG.md]: done.

**Review by silverwind — APPROVED** [review]

File: CHANGELOG.md [hunk 2]
```diff
@@ -4,6 +4,37 @@ This changelog goes through all the changes that have been made in each release
...
+  * Fix stats upon searching issues (#17566) (#17578)
+  * Escape issue titles in comments list (#17555) (#17556)
+  * Fix zero created time bug on commit api (#17546) (#17547)
+  * Fix database keyword quote problem on migration v161 (#17523)
+  * Fix email with + when active (#17518) (#17520)
+  * Stop double encoding blame commit messages (#17498) (#17500)
+  * Quote the table name in CountOrphanedObjects (#17487) (#17488)
+  * Run Migrate in Install rather than just SyncTables (#17475) (#17486)
+* BUILD
+  * Fix golangci-lint warnings (#17598 et al) (#17668)
```
Comment (6543) [inline CHANGELOG.md:34]: what is "et al" ?

Comment (lunny) [reply CHANGELOG.md:34]: Wow, that should be some typo. It's my cat? No, I have no cat. :)

Comment (jolheiser) [reply CHANGELOG.md:34]: It's Latin for "and others"

Comment (zeripath) [reply CHANGELOG.md:34]: There are a lot of PRs that that PR backports as one - too many to list - thence, et al.

File: CHANGELOG.md [hunk 3]
```diff
@@ -4,6 +4,37 @@ This changelog goes through all the changes that have been made in each release
 without substantial changes to our git log; to see the highlights of what has
 been added to each release, please refer to the [blog](https://blog.gitea.io).
 
+## [1.15.7](https://github.com/go-gitea/gitea/releases/tag/v1.15.7) - 2021-12-01
+
+* ENHANCEMENTS
+  * Only allow webhook to send requests to allowed hosts (backport #17482) (#17510)
```
Comment (zeripath) [inline CHANGELOG.md]:
```suggestion
  * Only allow webhook to send requests to allowed hosts (#17482) (#17510)
```

File: CHANGELOG.md [hunk 4]
```diff
@@ -4,6 +4,37 @@ This changelog goes through all the changes that have been made in each release
...
+  * Fix login redirection links (#17451) (#17473)
+* BUGFIXES
+  * Fix database inconsistent when admin change user email (#17840)
```
Comment (zeripath) [inline CHANGELOG.md]:
```suggestion
  * Fix database inconsistent when admin change user email (#17549) (#17840)
```

File: CHANGELOG.md [hunk 5]
```diff
@@ -4,6 +4,37 @@ This changelog goes through all the changes that have been made in each release
...
+  * Use correct user on releases (#17806) (#17818)
+  * Fix commit count in tag view (#17698) (#17790)
+  * Fix close issue but time watcher still running (#17761)
+  * Fix Migrate Description - backport (#17727)
```
Comment (zeripath) [inline CHANGELOG.md]:
```suggestion
  * Fix Migrate Description (#17692) (#17727)
```

File: CHANGELOG.md [hunk 6]
```diff
@@ -4,6 +4,37 @@ This changelog goes through all the changes that have been made in each release
...
+## [1.15.7](https://github.com/go-gitea/gitea/releases/tag/v1.15.7) - 2021-12-01
+
+* ENHANCEMENTS
+  * Only allow webhook to send requests to allowed hosts (backport #17482) (#17510)
+  * Fix login redirection links (#17451) (#17473)
+* BUGFIXES
+  * Fix database inconsistent when admin change user email (#17840)
+  * Use correct user on releases (#17806) (#17818)
+  * Fix commit count in tag view (#17698) (#17790)
+  * Fix close issue but time watcher still running (#17761)
```
Comment (zeripath) [inline CHANGELOG.md]:
```suggestion
  * Fix close issue but time watcher still running (#17643) (#17761)
```

File: CHANGELOG.md [hunk 7]
```diff
@@ -4,6 +4,37 @@ This changelog goes through all the changes that have been made in each release
...
+  * Return 400 but not 500 when request archive with wrong format (#17691) (#17700)
+  * Fix bug when read mysql database max lifetime (#17682) (#17690)
+  * Fix database deadlock when update issue labels (#17649) (#17665)
+  * Fix bug on detect issue/comment writer (#17592)
+  * Remove appSubUrl from pasted images (#17572) (#17588)
+  * Make `ParsePatch` more robust (#17573) (#17580)
+  * Fix stats upon searching issues (#17566) (#17578)
+  * Escape issue titles in comments list (#17555) (#17556)
+  * Fix zero created time bug on commit api (#17546) (#17547)
+  * Fix database keyword quote problem on migration v161 (#17523)
```
Comment (zeripath) [inline CHANGELOG.md]:
```suggestion
  * Fix database keyword quote problem on migration v161 (#17522) (#17523)
```

File: CHANGELOG.md [hunk 8]
```diff
@@ -4,6 +4,37 @@ This changelog goes through all the changes that have been made in each release
...
+  * Fix email with + when active (#17518) (#17520)
+  * Stop double encoding blame commit messages (#17498) (#17500)
+  * Quote the table name in CountOrphanedObjects (#17487) (#17488)
+  * Run Migrate in Install rather than just SyncTables (#17475) (#17486)
+* BUILD
+  * Fix golangci-lint warnings (#17598 et al) (#17668)
+* MISC
+  * Preserve color when inverting emojis (#17799)
```
Comment (zeripath) [inline CHANGELOG.md]:
```suggestion
  * Preserve color when inverting emojis (#17797) (#17799)
```

**Review by zeripath — APPROVED** [review]

**Review by 6543 — APPROVED** [review]

**Comment by 6543** [comment]
-> #17889

**Pull request merged**